contracts/access/manager/AccessManager.sol

```diff
@@ -139,8 +143,9 @@ contract AccessManager is Context, Multicall, IAccessManager {
             return (_isExecuting(target, selector), 0);
         } else {
             uint64 roleId = getTargetFunctionRole(target, selector);
-            (bool isMember, uint32 currentDelay) = hasRole(roleId, caller);
-            return isMember ? (currentDelay == 0, currentDelay) : (false, 0);
+            bool inRole;
+            (inRole, delay) = hasRole(roleId, caller);
+            return inRole ? (delay == 0, delay) : (false, 0);
```

Comment:
I agree with your point, I see it risky to have an assignment to the return value here, specially because multiple things around the contract depend on the delay returned being 0 if the user doesn't have the role. I'm updating